pages/index.vue

```diff
@@ -53,6 +62,12 @@
       },
       heroImage() {
         return this.hero ? this.hero.image.fields.file : null;
+      },
+      onHomePage() {
+        return Boolean(Number(process.env.ENABLE_LINKS_TO_CLASSIC)) && !this.path;
+      },
+      notificationUrl() {
+        return `https://classic.europeana.eu/${this.$store.state.i18n.locale}/portal/?utm_source=new-website&utm_medium=button`;
```

Comment:
This is incorrect. The locale comes after /portal, and there should not be another slash before `?`